doc/man3/OPENSSL_s390xcap.pod

```diff
@@ -159,7 +174,7 @@ Disables the KM-XTS-AES and and the KIMD-SHAKE function codes:
 
 =head1 SEE ALSO
 
-[1] z/Architecture Principles of Operation, SA22-7832-11
+[1] z/Architecture Principles of Operation, SA22-7832-12
```

Comment:
I assume this has not been released yet...

Reply:
right.